packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -410,6 +417,9 @@ class FloatingActionButton extends StatelessWidget {
     final Color hoverColor = this.hoverColor
       ?? floatingActionButtonTheme.hoverColor
       ?? theme.hoverColor;
+    final Color splashColor = this.splashColor
+        ?? floatingActionButtonTheme.splashColor
```

Comment:
2 space indent?

Reply:
Thanks for spotting, no idea how that happened!